test/infrastructure/docker/controllers/dockercluster_controller.go

```diff
@@ -70,9 +70,14 @@ func (r *DockerClusterReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 	}
 
 	log = log.WithValues("cluster", cluster.Name)
+	ipFamily, err := cluster.IPFamily()
+	if err != nil {
+		return ctrl.Result{}, err
+	}
+	ipv6 := ipFamily == clusterv1.IPv6Family
 
 	// Create a helper for managing a docker container hosting the loadbalancer.
-	externalLoadBalancer, err := docker.NewLoadBalancer(cluster.Name)
+	externalLoadBalancer, err := docker.NewLoadBalancer(cluster.Name, ipv6)
```

Comment:
nit: might be worth to consider passing the entire cluster object to `docker.NewLoadBalancer` instead of passing two info derived from cluster